src/components/controller/units/UnitsTable.vue

```diff
@@ -42,7 +51,17 @@ const props = defineProps({
   }
 })
 
-const emit = defineEmits(['reloadData', 'openSshModal'])
+// TODO: when dropdown nicely implements icons, remove this with correct imports directly in the component
```

Comment:
```suggestion
// FIXME: when dropdown nicely implements icons, remove this with correct imports directly in the component
```

Reply:
Worry not, a TODO will suffice, if we change the type of the icons the whole build will break :smile: